versions/1.0.0.md

```diff
@@ -613,52 +610,39 @@ components:
 }
 ```
 
-#### Reusable Parameter Object
+#### Reusable Object
 
-A simple object to allow reusable parameters from multiple steps or workflows in the Workflows Description.
+A simple object to allow referencing of objects contained within the [Components Object](#components-object). It can be leveraged from locations within steps or workflows in the Workflows Description. **Note** - Input Objects MUST use standard JSON Schema referencing via the `$ref` keyword while all non JSON Schema objects will leverage this object and it's expression based referencing mechanism.
 
 ##### Fixed Fields
 Field Name | Type | Description
 ---|:---:|---
-<a name="reusableParameterName"></a>name | `{expression}` | **REQUIRED**. A [runtime expression](#runtime-expressions) used to reference the desired parameter.
-<a name="reusableParameterValue"></a>value | `string` | A value by default MUST override that of the referenced parameter. 
+<a name="reusableObjectName"></a>reference | `{expression}` | **REQUIRED**. A [runtime expression](#runtime-expressions) used to reference the desired object.
+<a name="reusableObjectValue"></a>value | `string` | A value which by default MUST override that of the referenced parameter. This is only applicable for parameter object references.

```

Comment:
What does it mean "by default"? Are those words necessary? I wonder why this only applies to parameter object references?
```suggestion
<a name="reusableObjectValue"></a>value | `string` | Sets a value of the referenced parameter. This is only applicable for parameter object references.

```